resolver/resolver.cc

```diff
@@ -2554,7 +2554,14 @@ void Resolver::sanityCheck(core::GlobalState &gs, vector<ast::ParsedFile> &trees
 ast::ParsedFilesOrCancelled Resolver::runIncremental(core::GlobalState &gs, vector<ast::ParsedFile> trees) {
     auto workers = WorkerPool::create(0, gs.tracer());
     trees = ResolveConstantsWalk::resolveConstants(gs, std::move(trees), *workers);
-    computeLinearization(gs);
+    // NOTE: Linearization does not need to be recomputed as we do not mutate mixins() during incremental resolve.
+    if (debug_mode) {
+        for (auto i = 1; i < gs.classAndModulesUsed(); i++) {
```

Comment:
is this "all the classes and modules that had been added by this file" not "all symbols globally"?

Reply:
This is all symbols globally. We don't have a way to run symbol passes only over the subset of the symbol table that may have been impacted by the given files.

Reply:
do you think this debug mode check is going to make it like really long to debug on Stripe's codebase? should we make it respect skip_slow_enforce ?

Reply:
Nope. It's a linear walk of the symbol table checking a single bit per class. It should take a handful of ms I'd imagine.